Dockerfile.child_chain

```diff
@@ -62,6 +62,6 @@ EXPOSE $PORT
 # in inet_dist_listen_min and inet_dist_listen_max.
 #EXPOSE 4369 6900 6901 6902 6903 6904 6905 6906 6907 6908 6909
 
-ENTRYPOINT ["/init", "/entrypoint"]
+ENTRYPOINT ["/init", "/child_chain_entrypoint"]
```

Comment:
Where's this defined? And what's the longer term plan here? Is `launcher.py` going inside the start up mechanics somewhere? I'm thinking about local development and open source here.

Reply:
Entrypoints are defined in https://github.com/omisego/elixir-omg/tree/master/rootfs
`launcher.py` is going away, completely. Once the transition is over.
local development can then utilize a new docker-compose or the written Makefile